packages/contracts-bedrock/src/dispute/PermissionedDisputeGame.sol

```diff
@@ -31,6 +31,12 @@ contract PermissionedDisputeGame is FaultDisputeGame {
         _;
     }
 
+    /// @notice Semantic version.
+    /// @custom:semver 1.4.0
+    function version() public pure override returns (string memory) {
+        return "1.4.0";
```

Comment:
Should this be `super() + "-permissioned"`?

Reply:
I think they should be separate, in case the source of PDG changes so that they can change separately. 

I'm running a test now to ensure that a change to the FDG will cause an error if the PDG's semver isn't updated.

Reply:
OK, I managed to show [on a PR](https://github.com/ethereum-optimism/optimism/pull/14003/files) that builds on this one, that if you try to merge a change on the FDG into `develop`, then CI [will fail](https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/78955/workflows/8a60122b-2f4c-475e-bb1d-606137f0e7e4/jobs/3178848?invite=true#step-111-157_102) unless you also bump the semver on the PDG.

Reply:
Hmm ok I think this is reasonable. I'm not strongly opinionated. Only question to answer would be: why does this differ from +interop contracts where we have the same principle?

Reply:
The primary difference is that the interop contracts are not being released here, so I'm not concerned about updating them and wanted to keep the diff small. This also avoids concerning the interop devs, though I did run this by @tynes.

Once they have been released, then I would advocate for treating them similarly, ie. giving them their own independent semvers which must be incremented anytime their source or initcode  hashes change, regardless of where the source code change occcured.